### PR TITLE
Use fixed multipliers to generate Linea priority fees

### PR DESCRIPTION
**Pull request by matthewwalsh0**
## Explanation

Update the `LineaGasFeeFlow` to generate the priority fees using static multipliers.

Previously these were calculated by adding the differences between the priority estimates returned from the `GasFeeController`. However this could result in excessive increases between levels if the spread of priority fees grows due to additional pricing changes on the Linea nodes.

Also add a `gasFeeEstimatesLoaded` property to enable the clients to know when the first gas fee estimates update has completed.  

### `@metamask/transaction-controller`

- **ADDED**: Add `gasFeeEstimatesLoaded` property to `TransactionMeta`.
- **CHANGED**: Optimise calculation of `gasFeeEstimates` when creating transactions on Linea networks.

## Checklist

- [x] I've updated the test suite for new or updated code as appropriate
- [x] I've updated documentation (JSDoc, Markdown, etc.) for new or updated code as appropriate
- [x] I've highlighted breaking changes using the "BREAKING" category above as appropriate

### CODE REVIEW

File: packages/transaction-controller/src/gas-flows/LineaGasFeeFlow.ts [hunk 1]
```diff
@@ -46,10 +36,16 @@ const BASE_FEE_MULTIPLIERS = {
   high: 1.7,
 };
 
+const PRIORITY_FEE_MULTIPLIERS = {
+  low: 1,
+  medium: 1.05,
```
Comment (matthewwalsh0) [inline packages/transaction-controller/src/gas-flows/LineaGasFeeFlow.ts:41]: These values are pending confirmation from the Linea team.

File: packages/transaction-controller/src/gas-flows/LineaGasFeeFlow.ts [hunk 2]
```diff
@@ -129,64 +115,27 @@ export class LineaGasFeeFlow implements GasFeeFlow {
...
-      GasFeeEstimateLevel.high,
-      GasFeeEstimateLevel.medium,
-    );
-
-    const priorityFeeLow = hexToBN(lineaResponse.priorityFeePerGas);
-    const priorityFeeMedium = priorityFeeLow.add(mediumPriorityIncrease);
-    const priorityFeeHigh = priorityFeeMedium.add(highPriorityIncrease);
+    const low = hexToBN(value).muln(multipliers.low);
+    const medium = low.muln(multipliers.medium);
+    const high = low.muln(multipliers.high);
```
Comment (dbrans) [inline packages/transaction-controller/src/gas-flows/LineaGasFeeFlow.ts:130]: If `multipliers.low != 1` then I think `medium` and `high` will have an extra multiplier. 

```suggestion
    const base = hexToBN(value);
    const low = base.muln(multipliers.low);
    const medium = base.muln(multipliers.medium);
    const high = base.muln(multipliers.high);
```

**Review by dbrans — APPROVED** [review]

File: packages/transaction-controller/src/gas-flows/LineaGasFeeFlow.ts (hunk 2 above)
Comment (matthewwalsh0) [reply packages/transaction-controller/src/gas-flows/LineaGasFeeFlow.ts]: Great spot! Done.

**Comment by matthewwalsh0** [comment]
@metamaskbot publish-preview

File: packages/transaction-controller/src/gas-flows/LineaGasFeeFlow.ts (hunk 1 above)
Comment (matthewwalsh0) [reply packages/transaction-controller/src/gas-flows/LineaGasFeeFlow.ts:41]: Confirmed.

**Comment by matthewwalsh0** [comment]
@metamaskbot publish-preview

**Review by dbrans — APPROVED** [review]
Nice work!

**Review by OGPoyraz — APPROVED** [review]

**Pull request merged**